auto-capitalize keywords while typing

chaudum opened the pull request:
![crash-auto-capitalize](https://cloud.githubusercontent.com/assets/281260/22255966/7d98e580-e259-11e6-846b-f258fb0cadae.gif)

mikethebeer commented:
nice. However, I would like to have a setting that can be used to activate/deactivate this feature.

mikethebeer commented:
@chaudum noticed an issue that causes partly-capitalization of a word that is not registered as a keyword.

```sql
cr> SELECT SETtings FROM sys.cluster;
```

chaudum commented:
@mikethebeer addressed your comments
* added cli argument and runtime command: https://github.com/crate/crash/pull/169/commits/81853b71d30c1cd89a5cca9b3a513b913f6e831a
* improved capitalization: https://github.com/crate/crash/pull/169/commits/96d18f2fb85672f812bdd15c78d4ce97e30e9d56

/cc @Autophagy
